File: python/output.py
import time

inicio = time.time()

nome = 'Gustavo'
matricula = '12345'
curso = 'Sistemas de Informação'

print('O Aluno', nome, 'de matrícula', matricula, 'está cursando', curso)

tempoExec = time.time() - inicio
tempoExecMili = tempoExec * 1000

print("Tempo de execução: {} milissegundos".format(tempoExecMili))

File: cpp/output.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    
    setlocale(LC_ALL, "portuguese");
    
    string nome = "Gustavo";
    string matricula = "12345";
    string curso = "Sistemas de Informação";

    cout << "O Aluno " << nome << " de matrícula << " << matricula << " está cursando " << curso;
    
    return 0;
}

